randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -222,8 +222,7 @@ Extra - room_id: [6]
           Can Use Power Bombs
```

Comment:
For the sake of generator health, I would rather make this connection completely trivial, and put all the requirements that are needed to leave the item, on the way to the item.
So that door -> pb tank:
```md
ALL:
- can use any bomb
- ANY:
  - Grav and Damage run (bg) and Electric damage >= 20
  - Damage run (int) and Electric damage >= 80
```

Reply:
you want *all* of the requirements to just be on leaving? shouldn't the connection to the pickup at least have a requirement for pbs?

Reply:
Does the PB block respawn?

Reply:
no

Reply:
Also no, you misunderstood. I want *all* the rquirements on getting the item and want *none* on leaving.

Reply:
oh i see, my mistake